packages/@aws-cdk/aws-ecs-patterns/lib/base/network-load-balanced-service-base.ts

```diff
@@ -86,6 +86,15 @@ export interface NetworkLoadBalancedServiceBaseProps {
    */
   readonly loadBalancer?: NetworkLoadBalancer;
 
+  /**
+   * Listener port of the network load balancer that will serve traffic to the service.
+   *
+   * [disable-awslint:ref-via-interface]
```

Comment:
Not sure why you're disabling the linter here. Please remove this line.

Reply:
removed